schematics/web/janush/files/src/layouts/Logo/Logo.tsx.template

```diff
@@ -14,7 +14,7 @@ export const Logo: React.VFC = () => {
         data-testid="logo-button-in-top-app-bar"
         className={classes.logotype}
       >
-        <%= name %>
```

Comment:
It should stay

Reply:
I agree. My mistake. Thanks for remark ;)